### ISSUE TITLE
Time Filter - 3.2.3 Consistent Navigation - Only the up and down arrows can be used to prompt screen reader interpretation of dates

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10244]

**Comment by phudson-he** [comment]
I cannot alter the behavior of this because its a third party calendar plugin, so this would be marked as a "disproportionate burden" in your Accessibility Statement - The only fix would be to replace the calendar plugin with a more accessible calendar plugin
